src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -278,6 +282,48 @@ namespace AppInstaller::CLI::Workflow
             ReportInstallerResult("MsiInstallProduct"sv, APPINSTALLER_CLI_ERROR_MSI_INSTALL_FAILED);
     }
 
+    void PortableInstall(Execution::Context& context)
+    {
+        if (ExperimentalFeature::IsEnabled(ExperimentalFeature::Feature::PortableInstall))

```

Comment:
Do you need to check for the experimental feature here? 

`PortableInstall` is called from `ExecuteInstaller`, and since you added the check for the feature before the portable installer is ever downloaded, I believe this check is redundant?

Reply:
Removed redundant check